### PR TITLE
Quantization User Guide edits

### PR DESCRIPTION
**Pull request by dwelsch-esi**
Grammar and style edits to the Quantization User Guide.

### CODE REVIEW

File: Docs/user_guide/model_quantization.rst [hunk 1]
```diff
@@ -2,51 +2,51 @@
...
-of fixed precision parameters and fixed precision operations.
-
-AIMET provides multiple techniques and tools which help to create quantized models with a minimal loss in accuracy
-relative to floating-point models.
-
-This section provides information on typical use cases and AIMET's quantization features.
-
-Use Cases
+########################
+AIMET model quantization
```
Comment (quic-hitameht) [inline Docs/user_guide/model_quantization.rst:6]: Can we make the first character of each word capital in the titles?

Comment (dwelsch-esi) [reply Docs/user_guide/model_quantization.rst:6]: Al said to use the Microsoft style guide. Microsoft uses sentence-style capitalization in titles. We can discuss varying from that if it's important.

File: Docs/user_guide/quantization_aware_training.rst [hunk 2]
```diff
@@ -1,57 +1,51 @@
...
+#################################
 
 Overview
 ========
-In cases where PTQ techniques are not sufficient for mitigating quantization error, users can use quantization-aware
-training (QAT). QAT models the quantization noise during training and allows the model to find better solutions
-than post-training quantization. However, the higher accuracy comes with the usual costs of neural
-network training, i.e. longer training times, need for labeled data and hyperparameter search.
+
+When post-training quantizatio (PTQ) doesn't sufficiently reduce quantization error, the next step is to use quantization-aware training (QAT). QAT finds more accurate solutions than PTQ by modeling the quantization noise during training. This higher accuracy comes at the usual cost of neural network training, including longer training times and the need for labeled data and hyperparameter search.
```
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_aware_training.rst]: Typo: post-training quantizaito**n**

**Review by quic-jpolizzi — DISMISSED** [review]
This all looks good to me from a rst perspective. I would like to see a rendered example as well.

File: Docs/user_guide/model_guidelines.rst [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
+############################
 
-To implement the Cross Layer Equalization API,  aimet_torch.cross_layer_equalization.equalize_model(),  AIMET creates a computing graph to analyze the sequence of Operations in the model.
-If your model is defined using certain constructs, it restricts AIMET from successfully creating and analyzing the computing graph. The following table lists the potential issues and workarounds.
+To implement the Cross Layer Equalization API,  `aimet_torch.cross_layer_equalization.equalize_model()`,  AIMET creates a computing graph to analyze the sequence of operations in the model.
 
-Note: These restrictions are not applicable, if you are using the **Primitive APIs**
+Certain model constructs prevent AIMET from creating and analyzing the computing graph. The following table lists these potential issues and workarounds.
 
+.. admonition NOTE::
```
Comment (quic-hitameht) [inline Docs/user_guide/model_guidelines.rst]: This directive won't be rendered correctly, right? Either we should use specific admonition `.. note::` or generic admonition `..adomonition::`?  https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#admonitions-messages-and-warnings

File: Docs/user_guide/model_guidelines.rst [hunk 4]
```diff
@@ -1,28 +1,30 @@
...
 |                        | to export your model.        | specific layer so that ONNX       |
 |                        | Make sure ONNX export passes | export passes                     |
 +------------------------+------------------------------+-----------------------------------+
 | Slicing Operation      |Some models use               | Rewrite the x.view() statement    |
 |                        |**torch.tensor.view()** in the| as follows:                       |
-|                        |forward function as follows:  | x = x.view(x.size(0), -1)         |
-|                        |x = x.view(-1, 1024)          |                                   |
-|                        |If view function is written   |                                   |
-|                        |as above, it causes an issue  |                                   |
+|                        |forward function as follows:  | `x = x.view(x.size(0), -1)`       |
```
Comment (quic-hitameht) [inline Docs/user_guide/model_guidelines.rst]: Can we reference python function using `:func:` to have better readability?
For example, :func:`x.view()`, :func:`x.view(x.size(0), -1)`

File: Docs/user_guide/model_quantization.rst [hunk 5]
```diff
@@ -2,51 +2,51 @@
...
+    QAT enable you to fine-tune a model with quantization operations inserted in the network graph. In effect, it makes the model parameters robust to quantization noise.
+
+    Compared to PTQ, QAT requires a training pipeline and dataset and takes longer because it needs some fine-tuning, but it can provide better accuracy, especially at low bitwidths.
+    
+    A typical QAT workflow is illustrated here:
 
     .. image:: ../images/quant_use_case_2.PNG
 
-AIMET Quantization Features
+_aimet-quantization-features:
```
Comment (quic-hitameht) [inline Docs/user_guide/model_quantization.rst:47]: Does this page even exist? `_aimet-quantization-features:` Let's make sure before cross-referencing it.

File: Docs/user_guide/model_quantization.rst [hunk 6]
```diff
@@ -118,91 +115,78 @@ AIMET Quantization Features
...
-   :doc:`PyTorch Model Guidelines<../api_docs/torch_model_guidelines>`
+:doc:`PyTorch Model Guidelines<../api_docs/torch_model_guidelines>`
 
-    In the case of PyTorch, there exists the Model Validator utility, to automate the checking of certain PyTorch model
-    requirements, as well as the Model Preparer utility, to automate the updating of the model definition to align with
-    certain requirements.
+    PyTorch has two utilities to automate model complaince:
+    
+    - The Model Validator utility automates checking PyTorch model requirements
+    - he Model Preparer utility automates updating model definition to align with requirements
```
Comment (quic-hitameht) [inline Docs/user_guide/model_quantization.rst]: Typo here: `The`

File: Docs/user_guide/post_training_quant_techniques.rst [hunk 7]
```diff
@@ -2,14 +2,14 @@
...
+AIMET post-training quantization techniques
+###########################################
 
 Overview
 ========
 
-It is observed that some ML models show reduced inference accuracy when run on quantized hardware due to approximation noises. AIMET provides post-training quantization techniques that help adjust the parameters in the model such that the model becomes more quantization-friendly. AIMET post-training quantizations are designed to be applied on pre-trained ML models. These techniques are explained as part of the "Data-Free Quantization Through Weight Equalization and Bias Correction” paper at ICCV 2019 - https://arxiv.org/abs/1906.04721
+Sme ML models show reduced inference accuracy when run on quantized hardware due to approximation noise. AIMET provides post-training quantization techniques that help adjust the parameters in the model such that the model becomes more quantization-friendly. AIMET post-training quantizations are designed to be applied on pre-trained ML models. These techniques are explained as part of the "Data-Free Quantization Through Weight Equalization and Bias Correction” paper at ICCV 2019 - https://arxiv.org/abs/1906.04721
```
Comment (quic-hitameht) [inline Docs/user_guide/post_training_quant_techniques.rst]: Typo here: `Some`

File: Docs/user_guide/quantization_aware_training.rst (hunk 2 above)
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_aware_training.rst]: Suggestion: 

Change: "QAT finds more accurate solutions than PTQ by modeling the quantization noise during training"

To: "QAT finds more optimal solutions than PTQ by finetuning the model parameters in the presence of quantization noise"

File: Docs/user_guide/quant_analyzer.rst [hunk 8]
```diff
@@ -1,91 +1,87 @@
...
 AIMET QuantAnalyzer
-===================
+###################
 
 Overview
 ========
 
-The QuantAnalyzer feature analyzes the model for quantization and points out sensitive parts/hotspots in the model.
-The analyses are performed automatically, and only requires the user to pass in callbacks for performing forward pass and evaluation, and optionally a dataloader for MSE loss analysis.
+The QuantAnalyzer performs several analyses to identify sensitive areas and hotspots in the model. These analyses are performed automatically. To use QuantAnalyzier, you pass in callbacks to perform forward passes and evaluations, and optionally a dataloader for MSE loss analysis.
```
Comment (quic-hitameht) [inline Docs/user_guide/quant_analyzer.rst]: Typo in `QuantAnalyzier`

File: Docs/user_guide/quant_analyzer.rst [hunk 9]
```diff
@@ -1,91 +1,87 @@
...
+    - A user-defined function for passing labeled data through the model for evaluation, returning an accuracy metric
+    - (Optional, for running MSE loss analysis) A dataloader providing unlabeled data to be passed through the model
 
-**Note**: Typically on quantized runtimes, batch normalization layers will be folded where possible.
-So that users do not have to call a separate API to do so, QuantAnalyzer automatically performs Batch Norm Folding prior to running its analyses.
+Other quantization-related settings are also provided in the call to analyze a model.
+See :doc:`PyTorch QuantAnalyzer API Docs<../api_docs/torch_quant_analyzer>` for more about how to call the QuantAnalyzer feature.
 
-Detailed Analysis Descriptions
+.. admonition:: NOTE
```
Comment (quic-hitameht) [inline Docs/user_guide/quant_analyzer.rst]: Same comment - let's make sure this directive is rendered correctly.

File: Docs/user_guide/quant_analyzer.rst [hunk 10]
```diff
@@ -1,91 +1,87 @@
...
 
-4. Per layer statistics histogram:
-    Under the TF Enhanced quantization scheme, encoding min/max values for each quantizer are obtained by collecting a histogram of tensor values seen at that quantizer and potentially tossing out outliers.
+Per-layer statistics histogram
+    Under the TF Enhanced quantization scheme, encoding min/max values for each quantizer are obtained by collecting a histogram of tensor values seen at that quantizer and deleting outliers.
 
-    When this quantization scheme is selected, QuantAnalyzer will output plots for each quantizer in the model, displaying the histogram of tensor values seen at that quantizer.
-    These plots are available as part of the activations_pdf and weights_pdf folders, containing a separate .html plot for each quantizer.
+    When this quantization scheme is selected, QuantAnalyzer outputs plots for each quantizer in the model, displaying the histogram of tensor values seen at that quantizer.
+    These plots are available as part of the `activations_pdf` and `weights_pdf folders`, containing a separate .html plot for each quantizer.
```
Comment (quic-hitameht) [inline Docs/user_guide/quant_analyzer.rst]: Looks like you meant `weights_pdf`, right?

File: Docs/user_guide/quantization_configuration.rst [hunk 11]
```diff
@@ -39,53 +40,53 @@ When working with a new runtime with different rules, or for experimental purpos
...
-    Aside from the "ops" and "params" dictionary, additional settings governing quantizers in the model are available:
+    Outside the "ops" and "params" dictionaries, the following additional quantizer settings are available:
 
     - strict_symmetric:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", quantizers which are configured in symmetric mode will use strict symmetric quantization.
-        When set to "False" or omitting the parameter altogether, quantizers which are configured in symmetric mode will not use strict symmetric quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes quantizers configured in symmetric mode to use strict symmetric quantization.
+       "False", or omitting the parameter, causes quantizers configured in symmetric mode to not use strict symmetric quantization.
```
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_configuration.rst]: Change in indent here messes with the formatting

File: Docs/user_guide/quantization_configuration.rst [hunk 12]
```diff
@@ -39,53 +40,53 @@ When working with a new runtime with different rules, or for experimental purpos
...
 
     - unsigned_symmetric:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", quantizers which are configured in symmetric mode will use unsigned symmetric quantization when available.
-        When set to "False" or omitting the parameter altogether, quantizers which are configured in symmetric mode will not use unsigned symmetric quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes quantizers configured in symmetric mode use unsigned symmetric quantization when available.
+       "False", or omitting the parameter, causes quantizers configured in symmetric mode to not use unsigned symmetric quantization.
```
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_configuration.rst]: Same comment on indentation

File: Docs/user_guide/quantization_configuration.rst [hunk 13]
```diff
@@ -39,53 +40,53 @@ When working with a new runtime with different rules, or for experimental purpos
...
 
     - per_channel_quantization:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", parameter quantizers will use per channel quantization as opposed to per tensor quantization.
-        When set to "False" or omitting the parameter altogether, parameter quantizers will use per tensor quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes parameter quantizers to use per-channel quantization rather than per-tensor quantization.
+        When set to "False" or omitting the parameter, causes parameter quantizers to use per-tensor quantization.
```
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_configuration.rst]: Minor: Can make the phrasing here consistent with the other flags

File: Docs/user_guide/quantization_configuration.rst [hunk 14]
```diff
@@ -123,41 +124,39 @@ When working with a new runtime with different rules, or for experimental purpos
...
+                "False" places all quantizers of this op type in asymmetric mode.
+                Omitting the setting causes quantizers of this op type to fall back to the setting specified by the defaults section.
 
         - per_channel_quantization:
-            An optional parameter. If included, possible settings include "True" and "False".
-            When set to "True", parameter quantizers of this op type will use per channel quantization as opposed to per tensor quantization.
-            When set to "False", parameter quantizers of this op type will use per tensor quantization.
-            By omitting the setting, parameter quantizers of this op type will fall back to the setting specified by the defaults section.
+            Optional.  If included, value is "True" or "False".
+           "True" sets parameter quantizers of this op type to use per-channel quantization rather than per-tensor quantization.
```
Comment (quic-mtuttle) [inline Docs/user_guide/quantization_configuration.rst]: Same comment on indentation

File: Docs/user_guide/adaround.rst [hunk 15]
```diff
@@ -1,84 +1,81 @@
...
+By default, AIMET uses *nearest rounding* for quantization. A single weight value in a weight tensor is illustrated in the following figure. In nearest rounding, this weight value is quantized to the nearest integer value.
 
+The Adaptive Rounding (AdaRound) feature uses a subset of the unlabeled training data to adaptively round weights. In the following figure, the weight value is quantized to the integer value far from it.
 
 .. image:: ../images/adaround.png
     :width: 900px
 
-AdaRound Use Cases
-=====================
+AdaRound optimizes a loss function using the unlabelled training data to decide whether to quantize a weight to the closer or further integer value. AdaRound quantization acieves accuracy closer to the FP32 model using low bit-width integer quantization.
```
Comment (quic-hitameht) [inline Docs/user_guide/adaround.rst]: Typo: `Achieves`

Comment (quic-hitameht) [reply Docs/user_guide/adaround.rst]: Suggestion:

Change: "AdaRound quantization achieves accuracy closer to the FP32 model using low bit-width integer quantization."

To: "AdaRound quantization achieves accuracy closer to the FP32 model, while using low bit-width integer quantization"

File: Docs/user_guide/bn_reestimation.rst [hunk 16]
```diff
@@ -2,49 +2,44 @@
 
 
 ======================
-AIMET BN Re-estimation
+AIMET Batch Normal Re-estimation
```
Comment (quic-hitameht) [inline Docs/user_guide/bn_reestimation.rst]: Can we change this to `Batch norm Re-estimation`?

File: Docs/user_guide/quantization_aware_training.rst (hunk 2 above)
Comment (dwelsch-esi) [reply Docs/user_guide/quantization_aware_training.rst]: @quic-mtuttle I made this change, but used "better-optimized" rather than "more optimal". Question: What does "optimal" mean here if not "accurate"?

File: Docs/user_guide/adaround.rst (hunk 15 above)
Comment (dwelsch-esi) [reply Docs/user_guide/adaround.rst]: Changed.

File: Docs/user_guide/model_guidelines.rst (hunk 3 above)
Comment (dwelsch-esi) [reply Docs/user_guide/model_guidelines.rst]: Changed to ".. note::"

File: Docs/user_guide/model_guidelines.rst (hunk 4 above)
Comment (dwelsch-esi) [reply Docs/user_guide/model_guidelines.rst]: Reformatted the page, including code samples.

**Review by quic-hitameht — APPROVED** [review]

File: Docs/user_guide/model_quantization.rst (hunk 5 above)
Comment (dwelsch-esi) [reply Docs/user_guide/model_quantization.rst:47]: It's a reference within the same file.

File: Docs/user_guide/quantization_configuration.rst (hunk 11 above)
Comment (dwelsch-esi) [reply Docs/user_guide/quantization_configuration.rst]: Reformatted entire doc. Looks better now.

File: Docs/user_guide/quantization_configuration.rst (hunk 13 above)
Comment (dwelsch-esi) [reply Docs/user_guide/quantization_configuration.rst]: Fixed throughout the file.

File: Docs/user_guide/quantization_configuration.rst (hunk 14 above)
Comment (dwelsch-esi) [reply Docs/user_guide/quantization_configuration.rst]: Fixed.
